installers/vmware/kickstart-esxi.go

```diff
@@ -231,7 +231,7 @@ fi
 # Kickstart firstboot supplemental config URL
 if [ "$kickstartfburl" != "null" ]; then
 	echo "Using supplemental kickstart firstboot URL: $kickstartfburl"
-	if wget -q -s $kickstartfburl; then
+	if wget -q $kickstartfburl; then
 		wget -q $kickstartfburl -O /tmp/ks-firstboot-sup.sh
```

Comment:
Why download twice? This could just be
```suggestion
	if wget -q $kickstartfburl -O /tmp/ks-firstboot-sup.sh; then
```

Reply:
SGTM